pkgs/development/libraries/libevent/default.nix

```diff
@@ -20,6 +20,8 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  configureFlags = lib.optional (!sslSupport) [ "--disable-openssl" ];
```

Comment:
Since this is using `lib.optional` this results in `[ [ "--disable-openssl" ] ]`. Might as well remove the additional brackets.